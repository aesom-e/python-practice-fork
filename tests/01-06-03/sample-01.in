I do eat chickens
 not
4