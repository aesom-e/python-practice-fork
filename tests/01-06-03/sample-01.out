I do not eat chickens